lib/ui/painting/image_encoding.cc

```diff
@@ -132,19 +100,20 @@ void ConvertImageToRaster(sk_sp<SkImage> image,
   // to prevent concurrent usage of the image on both the IO and raster threads.
   raster_task_runner->PostTask([image, encode_task = std::move(encode_task),
                                 resource_context, snapshot_delegate,
-                                io_task_runner]() {
+                                io_task_runner, is_gpu_disabled_sync_switch]() {
     sk_sp<SkImage> raster_image =
         snapshot_delegate->ConvertToRasterImage(image);
 
     io_task_runner->PostTask([image, encode_task = std::move(encode_task),
                               raster_image = std::move(raster_image),
-                              resource_context]() mutable {
+                              resource_context,
+                              is_gpu_disabled_sync_switch]() mutable {
       if (!raster_image) {
         // The rasterizer was unable to render the cross-context image
         // (presumably because it does not have a GrContext).  In that case,
         // convert the image on the IO thread using the resource context.
-        raster_image =
-            ConvertToRasterUsingResourceContext(image, resource_context);
+        raster_image = ConvertToRasterUsingResourceContext<fml::SyncSwitch>(
```

Comment:
Remove `<fml::SyncSwitch>`, the type system will do that for you.

Reply:
Done